requirements/detection_test.txt

```diff
@@ -1,4 +1,4 @@
 # NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
-faster-coco-eval ==1.5.*
+faster-coco-eval >=1.6.2
```

Comment:
```suggestion
faster-coco-eval >=1.6.2, <1.7.0
```

Reply:
I'm already working on an implementation that will be stable for macOS, I think I'll release 1.6.3 today or tomorrow